#5 Fix for README example doesn't work.

Romick2005 opened the pull request:
Hi Rebecca,

Thank you for your work on html to rtf module.
I just used it and it it is cool, thank you.
Checked issues and decided to fix one. Please review and merge when you have time.

iarna commented:
PR your bug fix to rtf-parser please. I'm not going to make a dep on your git repo.=D

iarna commented:
Ah, I did take a commit to rtf-parser from you last July, maybe that was it?

Romick2005 commented:
Hi! Yes I couldn't wait so long for the fix to be included in your original repository so I switched to my forked copy to continue the work that's why I changed that in package.json. Yep there was 2 bugs 1) Readme example not work and probably it scared many people to use this module, but it was minor. 2) Problem with picking up correct font, that also was solved in my PR. Thanks for merging!
